frontend/components/Leaderboard.tsx

```diff
@@ -37,25 +43,37 @@ const Leaderboard: React.FC = () => {
   useEffect(() => {
     const fetchContributions = async () => {
       try {
-        const [topContributors, contributions] =
+        let [topContributors, contributions] =
           await burntPixArchives.getTopTenContributors();
         if (Number(contributions[0]) === 0) {
           return;
         }
+        const zeroContributionIndeces: number[] = [];
+        contributions.forEach((contribution, index) => {
+          if (Number(contribution) === 0) {
+            zeroContributionIndeces.push(index);
+          }
+        });
+        topContributors = topContributors.filter(
+          (_, index) => !zeroContributionIndeces.includes(index),
+        );
+        contributions = contributions.filter(
+          (_, index) => !zeroContributionIndeces.includes(index),
+        );
```

Comment:
Looking at https://github.com/yearone-io/burntpix-archives/blob/main/solidity/contracts/BurntPixArchives.sol#L230

I'm not quite sure how we can end up with contributors that have zero contributions

But if there is such case, would we be able to not return those from the smart contract? so that we don't have to do this extra logic here

Reply:
@democrazyfi when there not enough spots to fill 10. smart contract by default returns zero address w/ zero contributions for those slots